src/code/z_common_data.c

```diff
@@ -4,7 +4,8 @@ SaveContext gSaveContext;
 
 void SaveContext_Init(void) {
     bzero(&gSaveContext, sizeof(gSaveContext));
-    gSaveContext.playerForm = 0;
+
+    gSaveContext.save.playerForm = 0;
```

Comment:
Yeah, I think this specific place is setting this value to 0 instead of fierce deity (which just happens to be zero too).
In general it is very confusing and I could be wrong here too :s

Reply:
It's fine as is. This is something I've seen in other files too. Just something to keep in mind for later